circle.yml

```diff
@@ -702,6 +718,9 @@ workflows:
       - spectest:
           requires:
             - coverage-clang
+      - arm64:
+          requires:
+            - coverage-gcc
```

Comment:
Why coverage-gcc? That is the only gcc job we have?

Reply:
Arbitrary ordering. Both use GCC compiler.